google-api-client/pom.xml

```diff
@@ -104,6 +104,25 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.1.2</version>
+        <configuration>
+          <failOnWarning>true</failOnWarning>
+
+          <ignoredUsedUndeclaredDependencies>
+            <ignoredUsedUndeclaredDependency>org.apache.httpcomponents:httpcore</ignoredUsedUndeclaredDependency>
+            <ignoredUsedUndeclaredDependency>org.apache.httpcomponents:httpclient</ignoredUsedUndeclaredDependency>
+            <ignoredUsedUndeclaredDependency>com.google.http-client:google-http-client</ignoredUsedUndeclaredDependency>
+            <ignoredUsedUndeclaredDependency>com.google.protobuf:protobuf-java</ignoredUsedUndeclaredDependency>
+          </ignoredUsedUndeclaredDependencies>
+          <ignoredUnusedDeclaredDependencies>
+            <ignoredUnusedDeclaredDependency>commons-codec:commons-codec:jar:1.15</ignoredUnusedDeclaredDependency>
+          </ignoredUnusedDeclaredDependencies>
```

Comment:
This works; the build no longer fails at "dependencies (8)" at google-api-client module. But I'm not sure about the validity of these exceptions.